<!DOCTYPE html>
<html lang="en">
<head>
<script>


function myOpen() {
var demo = arguments[0];
  var x = document.getElementById(demo);
  if (x.style.display === 'none') {
    x.style.display = 'block';
  } else {
    x.style.display = 'none';
  }
}
</script>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="Research website of Behrooz Azarkhalili, a Ph.D.
  student at Stanford Enshallah!">
  <meta name="author" content="God">
  <link rel="shortcut icon" href="/img/favicon.ico" type="image/x-icon">
  <title>Behrooz Azarkhalili</title>

  <!-- CSS -->
        <link rel="stylesheet" href="https://a-slide.github.io/theme/css/bootstrap.flatly.min.css" type="text/css"/>
<link href="https://a-slide.github.io/theme/css/font-awesome.min.css" rel="stylesheet">

    <link href="https://a-slide.github.io/theme/css/pygments/native.css" rel="stylesheet">
    <link href="https://a-slide.github.io/theme/tipuesearch/tipuesearch.css" rel="stylesheet">
        <link href="https://a-slide.github.io/theme/css/typogrify.css" rel="stylesheet">
<link rel="stylesheet" href="https://a-slide.github.io/theme/css/style.css" type="text/css"/>
  <link rel="stylesheet" href="/css/main.css">
  <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
  <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
  <![endif]-->
</head>
<body>
  <div class="container">

    <div class="row" style="padding:20px">

       <div class="hidden-xs col-sm-3 col-md-2" id="sidebar" role="navigation" style="margin-top:180px">
        <hr>
        <ul class="nav nav-pills nav-stacked">
          <li><a href="#Research Interests">Research Interests</a></li>
          <li><a href="#Projects">Projects</a></li>
          <li><a href="#Contact Me">Contact Me</a></li>
          <li><a href="#publications">Publications</a></li>
          <li><a href="#Programming Languages">Programming Languages</a></li>
          <li><a href="#Technologies and Tools">Technologies and Tools</a></li>
          <li><a href="#IDEs and Text Editors">IDEs and Text Editors</a></li>
          <li><a href="#OS">OS</a></li>
        </ul>
      </div>

      <div class="col-xs-12 col-sm-9 col-md-9">

        <div class="row">
          <img src="img/babak.jpeg" class="pull-left" style="margin:20px
          20px 20px 0; height:140px; width:140px; border-radius:100%"/>
          <h2>Ermia (Behrooz) Azarkhalili</h2>
          <p class="lead">
            Data Scientist at Royan Institute<br>
            <br>

            <a href="https://twitter.com/ErmiaBivatan" class="icon">
              <i class="fa fa-twitter fa-lg"></i>
            </a>
            <a href="https://github.com/ErmiaAzarkhalili" class="icon">
              <i class="fa fa-github fa-lg"></i>
            </a>
          </p>
        </div>

        <hr>

        <div class="row" align="justify" style="font-size:14px;">
          <p>
          I am a data scientist at
          <a href="http://royaninstitute.org/">Royan Institute</a>. conducting research on the applications of deep and
machine learning algorithms in bioinformatics subdisciplines like cancer detection, drug
discovery, drug repositioning, personalized medication, and electronic health records. 
          </p>
<p></p>
          <p>
My particular interest is to develop computationally efficient DL methods to resolve problems arising in noisy and high dimensional datasets. The fundamental problem here is to infer reliable conclusions from noisy observations or data, specifically when limited computations and data are available. Collectively with modern settings where the dimensionality of data is high, achieving these twin objectives simultaneously is intricate. It appears that one of the current trends in DL is to resolve the aforementioned problems by implementing probabilistic and statistical modeling techniques. To become an expert in these fields, it is quintessential to possess deep foundations in Bayesian Deep Learning and Generative Probabilistic Models, which are among my main research topics.
</p>
<p></p>
<p>Alongside with Deep Probabilistic Models, I am immensely interested in understanding and developing de-novo Transfer Learning techniques such as Multi-Task Learning (MTL), Sequential Transfer Learning, and Deep Domain Adaptation in NLP and CV, which, for instance, can be applied to question answering (QA), neural machine translation (NMT), object detection and object segmentation. Furthermore, I am enthusiastic to utilize and develop novel attention based on recurrent network models alongside deep contextualized word representations (such as ELMO, BERT, and Transformers) on image or video caption generation for movies scene interpretation or inference from scenes.
</p>
<p></p>
<!--<p>
Currently, I have remotely joined to          
 <a href="https://www.genapsys.com/">GenapSys</a>
           to effectively resolve the DNA sequencing problem by implementing state-of-the-art algorithms of neural networks. Providing the python package to easily implement the DNA sequencing, the primary goal of the project is to revolutionize the DNA sequencing process.
          </p> -->

<p></p>
<p>
I am looking forward to a PhD position in outstanding labs in the USA or Europe. Please kindly ping me if you are intersted in my resume and research background.
</p>
<p>
        <!--  Recently, I have been giving the following talk:
          </p>
          <ul>
            <li>
              <div class="btn-group-xs">
                <strong>What Might Deep Learners Learn From Probabilistic
                  Programming?</strong>
                <a href="/talks/Tran_Probabilistic_Programming.pdf"
                class="btn btn-default">Slides</a>
              </div>
            </li>
          </ul>-->

        </div>
        <div class="row">
          <h3><a name="Research Interests"></a>Research Interests</h3>
          
            <ul style="list-style-type:square;">
                <strong><li>Natural Language Processing (NLP)</li></strong>
                <strong><li>Computrer Vision (CV)</li></strong>
                <strong><li>Deep Probabilistic Learning</li></strong>
                <strong><li>Bioinfromatics</li></strong>
            </ul>
          </p>
        </div>
        <div class="row" align="justify">
          <h3><a name="Projects"></a>Projects</h3>
          <p>
            <ul style="list-style-type:square;font-size:14px;">
                <p>Since 2016, I have joined Royan Institute for Computational Biology as a data scientist to work under the supervision of Dr. Ali Sharifi Zarchi, the member of Scientific Committee in International Informatics Olympiad (IoI) and the Professor of CS department at Sharif University of Technology. At Royan, my efforts were to apply DL techniques to cancer detection problems which have resulted in two first-author papers.</p><li><h4>Status: Active</h4><p style="font-size:14px;"><strong>Understanding Cell Identity Using Deep Neural Networks.</strong><div class="btn-group-xs"><a onclick="myOpen('demo9')" href="javascript:void(0);"
              class="btn btn-default">Detail</a>

             <div  id="demo9" style="display:none;font-size:14px;"><p>
I was the co-first author of the paper <a href="https://www.nature.com/articles/s41598-019-38798-y.">[1]</a>, ‚Äù Cell Identity Codes: Understanding Cell Identity from Gene Expression Profiles using Deep Neural Networks ‚Äù, which was accepted in Nature Scientific Reports. In this project, we identified a specific structure of deep autoencoders that can encode gene expression profiles (GEP) into a vector of 30 numeric values, which we called the cell identity code (CIC). We also showed that the distinctive values of the CIC are connected to various biological aspects of the cell, such as distinct pathways or biological processes. My contribution was to implement the python code project, analyzing the data and results, and writing the main draft of the paper. In this project, I utilized Tensorflow, Keras, Pytorch, Scikit Learn, and HyperOpt as the main packages to develop the source code.</p></div></div></p></li>
<li><h4>Status: Active</h4><p style="font-size:14px;"><strong>Cancer Detection Using Deep Learning.</strong><div class="btn-group-xs"><a onclick="myOpen('demo8')" href="javascript:void(0);"
              class="btn btn-default">Detail</a>

             <div  id="demo8" style="display:none;font-size:14px;"><p>
In the second paper <a href="https://arxiv.org/abs/1808.02237">[2]</a>, ‚Äù DeePathology: Deep Multi-Task Learning for Inferring Molecular Pathology from Cancer Transcriptome ‚Äù, which was submitted to ‚ÄùNature Scientific Reports‚Äù, we introduced a novel architecture of DNNs that was capable of simultaneous inference of various properties of biological samples, through multi-task and transfer learning. Our methodology significantly outperforms prior works and classical machine learning approaches in predicting tissue-of-origin, normal or disease state and cancer type of each sample. We also showed that this system is remarkably robust against noise and missing values. Besides writing the main draft of the paper, I developed a Python source code based on Tensorflow and Keras to resolve this Multi-Task learning problem.</p></div></div></p></li>
<li><h4>Status: In progress</h4><p style="font-size:14px;"><strong>Prediction of Clinical Events Using Deep Contextualized Word Embedings.</strong><div class="btn-group-xs"><a onclick="myOpen('demo7')" href="javascript:void(0);"
              class="btn btn-default">Detail</a>

             <div  id="demo7" style="display:none;font-size:14px;"><p>

I have recently embarked on with Dr. Sharifi Zarchi where we are applying recent innovative NLP concepts and tools including Attention, BERT, GPT2, and ELMO on biomedical and clinical notes to conduct a systematic review of deep learning models for electronic health record (EHR) data, and illustrate various deep learning architectures for analyzing different data sources and their target applications including information extraction and sequential prediction of clinical events. This work resulted in achieving the SOTA results in survival analysis of different cancer types.
</p></div></div>
</br></br>
<p>I have recently embarked on my research with Dr. Mohammad Amin Sadeghi, who has completed his Ph.D. at the CS department at UIUC, in the Computer Vision Laboratory at Tehran University. </p>
</p></li>
<li>
  <h4>Status: In progress</h4>
<p style="font-size:14px;"><strong>Dollar Street: Prediction of Individuals' Income Utilizing the Images of Their Posession.</strong></p><div class="btn-group-xs">
<a onclick="myOpen('demo7')" href="javascript:void(0);" class="btn btn-default">Detail</a>
 <div  id="demo7" style="display:none;font-size:14px;"><p>
</br>In this project, we have been working on the project of predicting income of individuals based on the images of their commodities and possessions. To solve this problem, we are utilizing de nova techniques such as transfer learning, Deep Set, Dimensionality Reduction, Ensemble Methods, and Hyper-Parameter Optimization to estimates families‚Äô income based on the images of their commodities. I was one of the main developer of the project where I implemented python code utilizing Keras, Tensorflow, Scikit Learn, MLEns, and HyperOpt.</p></div></div>
</li>
            </ul>

        </div>
     <div>
<h3><a name="Contact Me"></a> Contact Me</h3> 
<p align="left"> <a href="https://twitter.com/Azarkhalili" target="_blank"> <img src="https://img.shields.io/badge/Twitter-282C34?logo=twitter" alt="Twitter logo" title="Twitter" height="25" /> </a> <a href="mailto:ermiaazarkhalili@gmail.com?"> <img src="https://img.shields.io/badge/Gmail-282C34?logo=gmail" alt="Gmail logo" title="Gmail" height="25" /> </a> </p>



<h3><a name="Publications"></a> Publications</h3>
<p align="left"> <a href="https://scholar.google.com/citations?user=Jid12csAAAAJ&hl=en" target="_blank"> <img src="https://img.shields.io/badge/Google Scholar-282C34?logo=google-scholar&logoColor=4D90FE" alt="Google Scholar logo" title="Google Scholar" height="25" /> </a> <a href="https://arxiv.org/search/stat?searchtype=author&query=Azarkhalili%2C+B" target="_blank"> <img src="https://img.shields.io/badge/Arxiv-282C34?logo=arxiv&logoColor=red" alt="Arxiv logo" title="Arxiv" height="25" /> </a> </p>


<h3><a name="Programming Languages"></a> Programming Languages</h3>
<p align="left"> <a href="https://python.org/" target="_blank"> <img src="https://img.shields.io/badge/Python-282C34?logo=Python&logoColor=2BA1B9" alt="Python logo" title="Python" height="25" /> </a> <a href="https://https://www.r-project.org/" target="_blank"> <img src="https://img.shields.io/badge/R-282C34?logo=R&logoColor=6F9BBC" alt="R logo" title="R" height="25" /> </a> <a href="https://scala-lang.org/" target="_blank"> <img src="https://img.shields.io/badge/Scala-282C34?logo=Scala&logoColor=DC322F" alt="Scalalogo" title="Scala" height="25" /> </a> <a href="https://www.mathworks.com/" target="_blank"> <img src="https://img.shields.io/badge/Mathworks-282C34?logo=mathworks&logoColor=F78F10" alt="Matlab logo" title="Matlab" height="25" /> </a> <a href="https://www.latex-project.org/" target="_blank"> <img src="https://img.shields.io/badge/Latex-282C34?logo=latex&logoColor=AFCB1E" alt="Latex logo" title="Latex" height="25" /> </a> <a href="https://www.gnu.org/software/octave" target="_blank"> <img src="https://img.shields.io/badge/Octave-282C34?logo=octave" alt="Octave logo" title="Octave" height="25" /> </a> <a href="https://www.markdownguide.org/cheat-sheet/" target="_blank"> <img src="https://img.shields.io/badge/Markdown-282C34?logo=markdown&logoColor=93C6F4" alt="Markdown logo" title="Markdown" height="25" /> </a> </p>

<h3><a name="Technologies and Tools"></a> üõ†¬†¬†Technologies and Tools</h3>
<p align="left"> <a href="https://tensorflow.org/" target="_blank"> <img src="https://img.shields.io/badge/Tensorflow-282C34?logo=tensorflow" alt="Tensorflow logo" title="Tensorflow" height="25" /> </a> <a href="https://pyotrch.org/" target="_blank"> <img src="https://img.shields.io/badge/Pytorch-282C34?logo=pytorch&style=flat" alt="Pytroch logo" title="Pytorch" height="25" /> </a> <a href="https://keras.io/" target="_blank"> <img src="https://img.shields.io/badge/Keras-282C34?logo=keras&logoColor=D00000" alt="Keras logo" title="Keras" height="25" /> </a> <a href="https://sklearn.org/" target="_blank"> <img src="https://img.shields.io/badge/Scikit Learn-282C34?logo=scikit-learn" alt="ScikitLearn logo" title="Scikit Learn" height="25" /> </a> <a href="https://git-scm.com/" target="_blank"> <img src="https://img.shields.io/badge/Git-282C34?logo=git" alt="Git logo" title="Git" height="25" /> </a>  <a href="https://github.com/" target="_blank"> <img src="https://img.shields.io/badge/GitHub-282C34?logo=github" alt="GitHub logo" title="GitHub" height="25" /> </a> <a href="https://gitlab.com/" target="_blank"> <img src="https://img.shields.io/badge/GitLab-282C34?logo=gitlab" alt="GitLab logo" title="GitLab" height="25" /> </a> <br> <a href="https://spark.apache.org/" target="_blank"> <img src="https://img.shields.io/badge/Apache Spark-282C34?logo=apache-spark" alt="Apache Spark logo" title="Apache Spark" height="25" /> </a>  <a href="https://airflow.apache.org/" target="_blank"> <img src="https://img.shields.io/badge/Apache Airflow-282C34?logo=apache-airflow&logoColor=AFCB1E" alt="Apache Airflow logo" title="Apache Airflow" height="25"/> </a> <a href="https://www.heroku.com/" target="_blank"> <img src="https://img.shields.io/badge/Heroku-282C34?logo=heroku&logoColor=A3AAEB" alt="Heroku logo" title="Heroku" height="25" /> </a> <a href="https://streamlit.io/" target="_blank"> <img src="https://img.shields.io/badge/Streamlit-282C34?logo=streamlit" alt="Streamlit logo" title="Heroku" height="25" /> </a> <a href="https://www.mongodb.com/" target="_blank"> <img src="https://img.shields.io/badge/MongoDB-282C34?logo=mongodb" alt="MongoDB logo" title="MongoDB" height="25" /> </a> <a href="https://www.sqlite.org/" target="_blank"> <img src="https://img.shields.io/badge/SQLite-282C34?logo=sqlite&logoColor=64AEDC" alt="SQLite logo" title="SQLite" height="25"/> </a> <a href="https://www.postgresql.org/" target="_blank"> <img src="https://img.shields.io/badge/PostgreSQL-282C34?logo=postgresql&logoColor=64AEDC" alt="PostgreSQL logo" title="PostgreSQL" height="25"/> </a> <a href="https://www.wakatime.com/" target="_blank"> <img src="https://img.shields.io/badge/WakaTime-282C34?logo=wakatime&logoColor=F75000" alt="WakaTime logo" title="WakaTime" height="25"/> </a> <a href="https://www.clickup.com/" target="_blank"> <img src="https://img.shields.io/badge/ClickUp-282C34?logo=clickup" alt="ClickUp logo" title="ClickUp" height="25"/> </a> </p>

<h3><a name="IDEs and Text Editors"></a> IDEs and Text Editors</h3>
<p align="left">  </a><a href="https://code.visualstudio.com/" target="_blank"> <img src="https://img.shields.io/badge/Visual Studio Code-282C34?logo=visual-studio-code&logoColor=ffffff" alt="VS Code logo" title="VS Code" height="25" /> </a> <a href="https://https://www.jetbrains.com/pycharm/" target="_blank"> <img src="https://img.shields.io/badge/Pycharm-282C34?logo=pycharm" alt="Pycharm logo" title="Pycharm" height="25" /> </a> <a href="https://www.jetbrains.com/idea/" target="_blank"> <img src="https://img.shields.io/badge/Intellij IDEA-282C34?logo=intellij-idea" alt="Intellij IDEA logo" title="Intellij IDEA" height="25" /> </a> <a href="https://rstudio.com" target="_blank"> <img src="https://img.shields.io/badge/RStudio-282C34?logo=rstudio&logoColor=ffffff" alt="RStudio logo" title="RStudio" height="25" /> </a> </a> <a href="https://colab.google.com" target="_blank"> <img src="https://img.shields.io/badge/Google Colab-282C34?logo=google-colab&logoColor=ffffff" alt="GoogleColab logo" title="Google Colab" height="25" /> </a> <a href="https://jupyter.org/" target="_blank"> <img src="https://img.shields.io/badge/Jupyter-282C34?logo=jupyter&logoColor=ffffff" alt="Jupyter logo" title="Jupyter" height="25" /> </a>  </a></p>


<h3><a name="OS"></a> OS</h3>
<p align="left">  </a> <a href="https://ubuntu.com/" target="_blank"> <img src="https://img.shields.io/badge/Ubuntu-282C34?logo=ubuntu" alt="Ubuntu logo" title="Ubuntu" height="25" /> </a> <a href="https://linuxmint.com/" target="_blank"> <img src="https://img.shields.io/badge/Linux Mint-282C34?logo=linux-mint" alt="Mint logo" title="Mint" height="25" /> </a> <a href="https://www.debian.org/" target="_blank"> <img src="https://img.shields.io/badge/Debian-282C34?logo=debian&logoColor=D0074E" alt="Debian logo" title="Debian" height="25" /> </a> </p>

  </div>


  <!-- JavaScript -->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>
  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        inlineMath: [ ['$','$'] ],
        processEscapes: true
      }
    });
  </script>
  <script type="text/javascript"
     src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  </script>
  <script type="text/javascript" src="/js/main.js"></script>
</body>
</html>
